src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -488,6 +488,36 @@ public String getDefaultBranch() throws IOException, InterruptedException {
         return getBranchesByRef("/refs/branches");
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @NonNull
+    public List<BitbucketCloudBranch> getBranchesByFilterText(String filterText) throws IOException, InterruptedException {
+        ArrayList<BitbucketCloudBranch> branches = new ArrayList<>();
+        for (BitbucketCloudBranch branch : getBranches()) {
+            if (branch.getName().contains(filterText)) {
```

Comment:
what happens if filterTest is something like `feature/*` ?

Reply:
yes, that is what i suspected, a first try would be to return all branches like before

Reply:
I believe our `fitlerText` variable here would be set to `master PR-*` which has always seemed to work for us. Guessing that in this case the check is a lot more blunt than expected given the specification of the filter text field?

Reply:
we have similar filter `master support/* PR-*`

Reply:
filterText can only be a dedicated branch name as it is called with scmHead.getName(), maybe there is some escaping in the names